src/test/java/teammates/test/cases/browsertests/InstructorCourseDetailsPageUiTest.java

```diff
@@ -239,14 +237,14 @@ private InstructorCourseDetailsPage getCourseDetailsPage() {
         return loginAdminToPage(detailsPageUrl, InstructorCourseDetailsPage.class);
     }
 
-    private boolean didStudentReceiveReminder(String courseName, String courseId, String studentEmail,
-                                              String studentPassword)
+    // TODO replace with GAE LocalMailService stub as we are checking the logic here, not reliability of email delivery
```

Comment:
Hmm I see. I just don't really like how the test can fail so easily. For example, emails may take up to a minute to receive.